server/controllers/userController.js

```diff
@@ -0,0 +1,268 @@
+const User = require('../models/userModel.js');
+const Group = require('../models/groupModel.js');
+const jwt = require('jsonwebtoken');
+const encryptPassword = require('../helpers/encryptPassword.js');
+const axios = require('axios');
+const {OAuth2Client} = require('google-auth-library');
+const client = new OAuth2Client(process.env.GOOGLE_CLIENT_ID);
+
+class UserController {
+    static register(req, res) {
+        let user = new User({
+            username: req.body.username,
+            email: req.body.email,
+            password: req.body.password
+        });
+        user.save()
+            .then(function(user) {
+                const response = {
+                    success: true,
+                    message: `Account ${user.username} registered`
+                }
+                res.status(201).json(response);
+            })
+            .catch(function(err) {
+                res.status(500).json(err);
+            });
+    }
+
+    static login(req, res) {
+        req.body.password = encryptPassword(req.body.password);
+
+        User.findOne({
+            email: req.body.email,
+            password: req.body.password
```

Comment:
Kenapa harus disimpan jadi sebuah variable?

```suggestion
 User.findOne({
            email: req.body.email,
            password: encryptPassword(req.body.password)
 })
 ```